src/Item_Project.php

```diff
@@ -265,16 +267,97 @@ public function getTabNameForItem(CommonGLPI $item, $withtemplate = 0)
 
     public static function displayTabContentForItem(CommonGLPI $item, $tabnum = 1, $withtemplate = 0)
     {
+        /** @var array $CFG_GLPI */
+        global $CFG_GLPI;
 
         switch ($item->getType()) {
             case 'Project':
                 self::showForProject($item);
                 break;
 
             default:
-               // Not defined and used now
-               // Project::showListForItem($item);
+                if (in_array($item->getType(), $CFG_GLPI["project_asset_types"])) {
+                    self::showForAsset($item);
+                }
+                break;
         }
         return true;
     }
+
+    public static function showForAsset(CommonDBTM $item)
+    {
+
+        $item_project = new self();
+        $item_projects = $item_project->find([
+            'itemtype' => $item->getType(),
+            'items_id' => $item->getID()
+        ]);
+
+        $used = $entries = [];
+
+        foreach ($item_projects as $value) {
+            $used[] = $value['projects_id'];
+            $project = new Project();
+            $project->getFromDB($value['projects_id']);
+
+            $priority = CommonITILObject::getPriorityName($project->fields['priority']);
+            $state = ProjectState::getById($project->fields['projectstates_id']);
+
+            if (!empty($project->fields)) {
+                $data = [
+                    'name' => $project->getLink(),
+                    'projectstates_id' => $state !== false
+                            ? sprintf(
+                                '<div class="badge_block" style="border-color:%s"><span class="me-1" style="background:%s"></span>%s',
+                                htmlescape($state->fields['color']),
+                                htmlescape($state->fields['color']),
+                                htmlescape($state->fields['name']),
+                            )
+                            : '',
+                    'priority' => sprintf(
+                        '<div class="badge_block" style="border-color: #ffcece"><span class="me-1" style="background: #ffcece"></span>%s',
+                        htmlescape($priority)
+                    ),
+                    'percent_done' => $project->fields['percent_done'] . '%',
+                ];
+                $entries[] = array_merge($project->fields, $data);
```

Comment:
This would include too much data and security issues could be difficult to foresee with this type of code. Please pass only required data to the template.

Also, IMHO, if the user cannot see the project (`!$project->can($project->fields['id'], READ)`), only its name should be available.